ncm-spma/src/main/perl/spma/yumng.pm

```diff
@@ -0,0 +1,810 @@
+# #
+# Software subject to following license(s):
+#   Apache 2 License (http://www.opensource.org/licenses/apache2.0)
+#   Copyright (c) Responsible Organization
+#
+
+# #
+# Current developer(s):
+#   Jindrich Novy <jindrich.novy@morganstanley.com>
+#
+
+# #
+# Author(s): Jindrich Novy
+#
+
+package NCM::Component::spma::yumng;
+
+#
+# a few standard statements, mandatory for all components
+#
+use strict;
+use warnings;
+use NCM::Component;
+our $EC  = LC::Exception::Context->new->will_store_all;
+our @ISA = qw(NCM::Component);
+use EDG::WP4::CCM::Element qw(unescape);
+
+use CAF::Process;
+use CAF::FileWriter;
+use Set::Scalar;
+use File::Path qw(mkpath rmtree);
+use LWP::Simple;
+use Text::Glob qw(match_glob);
+
+use constant HOSTNAME            => "/system/network/hostname";
```

Comment:
would be nice to be able to indentify the common ones with "regualr" yum, instead of redefining them.